### PR TITLE
 [Edit] CSS [Edit] CSS: Filter Functions sepia #4869

### LINKED ISSUE #4869: [Edit] CSS [Edit] CSS: Filter Functions `sepia`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `sepia` term entry under Filter Functions concept in CSS. The entry lives in `content/css/concepts/filter-functions/terms/sepia/sepia.md`.

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a output image comparing the original image in the example already given in the entry to its sepia filter version.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by JEllis1313**
### Description

-Added a output image comparing the original image in the example already given in the entry to its sepia filter version.
### Issue Solved

  
-Closes #4869 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [ x] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: docs/content/css/concepts/filter-functions/terms/sepia/sepia.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Code Example
+
+```html
+<img src="images/original-image.jpg" alt="Original Image" class="original">
+<img src="images/original-image.jpg" alt="Sepia Filtered Image" class="sepia">
+
+.sepia {
+    filter: sepia(1%);
+} 
+```
```
Comment (ishg-153) [inline docs/content/css/concepts/filter-functions/terms/sepia/sepia.md:62]: It is not necessary to add this as there is one example code given above. You just need to add output image for it.

File: docs/content/css/concepts/filter-functions/terms/sepia/sepia.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The `sepia` function converts the input image to sepia tones. This effect is achieved by applying a sepia tone to the image, giving it a warm brown tint reminiscent of early photography.
+
+## Example
+
+### Original Image
+![Original Image](images/original-image.jpg)
+
+### Sepia Filtered Image
+![Sepia Filtered Image](images/sepia-image.jpg)
```
Comment (ishg-153) [inline docs/content/css/concepts/filter-functions/terms/sepia/sepia.md:51]: You haven't added these images to the media folder. Please follow the directions here:
https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@JEllis1313 You seem to have created a new entry. Please ensure that you are making changes to the existing entry at `content/css/concepts/filter-functions/terms/sepia/sepia.md`. Also, you need to delete the `CSS-filter_Functions-Sepia` submodule that you have added to this repo.

Then make the following changes.

**Comment by mamtawardhani** [comment]
Hey @JEllis1313 are you still working on this? Note: If you do not revert back within 3 days this PR will be closed

**Comment by Sriparno08** [comment]
Closing this PR since the author is inactive.

**Pull request closed without merging**